### DISCUSSION TITLE
Why JsUnknown, JsExternal not impl Clone and Copy trait?

### DISCUSSION THREAD

**Discussion by SyMind** [napi-rs/napi-rs#2276] [Q&A]
Why JsUnknown, JsExternal not impl Clone and Copy trait?

**Comment by Brooooooklyn** [comment]
`JsUnknown` and `JsExternal` is about to be refactored and add Lifetime like [`Function`](https://github.com/napi-rs/napi-rs/blob/napi%403.0.0-alpha.12/crates/napi/src/bindgen_runtime/js_values/function.rs#L93)

All `JsXXX` should have a lifetime attached to the `#[napi] fn` and cannot be copy or clone because they would be invalid outside the function call scope.
